vcsaddons/Polar_1D/Polar_1d.ipynb

```diff
@@ -77,12 +74,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Plotting data\n",
+    "# Plotting Data\n",
     "\n",
     "\n",
     "[Back to Top](#top)\n",
     "\n",
-    "There are seven preconfigured polar oned plots provided with VCSAddons:\n",
+    "There are seven preconfigured polar one-d plots provided with VCSAddons:\n",
```

Comment:
Here is the 'one-d' I mentioned. You could change it to '1-d' maybe?

Reply:
Thanks @downiec. Good suggestion. I took a closer look at the text and made several edits as well as changing the one-d/1-d to 1D. I also talked to @doutriaux1 about the line of code that was commented out because it wasn't working, it seems best to just remove that line and others related to it. Since I've made significant changes to the notebook, could you review it again, @downiec?